build/package/scripts/build-typescript.sh

```diff
@@ -56,8 +70,24 @@ if [ -n "${NEXUS_HOST}" ] && [ -n "${NEXUS_USERNAME}" ] && [ -n "${NEXUS_PASSWOR
     npm config set strict-ssl=false
 fi;
 
-echo "Building ..."
+echo "Installing dependencies ..."
 npm ci
+
+echo "Linting ..."
+set +e
+npx eslint src --ext "${LINT_FILE_EXT}" --format compact --max-warnings "${MAX_LINT_WARNINGS}" > eslint-report.txt
```

Comment:
Maybe one might also be flexible with another output format than `compact`, what do you think?

Reply:
There is this question raised by @michaelsauter #216 of which I like the idea of. Therefore I thought it would be good to have a consistent output/format of the report and not to allow formatting to JSON or XML - any opinions?

Reply:
Is there a way to have two outputs? I assume `compact` to be somewhat readable, but JSON would be nice as one could inspect it later on more easily in tooling.

Reply:
Looking at documentation, this is not possible, but we could run `eslint` twice with different a different output flag. The linting stage usually does not take longer than 10 seconds for huge workspaces (this might even be exaggerated as I base this duration on experiences only), so we *could* decide to run it twice in order to also have a linting artifact in JSON without a major increase in the duration of the pipeline.

Reply:
😞 

I would not run it twice until we have a concrete use case why we need both. For now let's stay with `compact` then.